lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/robot.cpp

```diff
@@ -25,7 +25,7 @@ void update(int32_t &act_right_curr) {
 }
 
 void cb(int8_t lin_act_volt) {
-  act_left_mtr.write(-lin_act_volt);
+  act_left_mtr.write(lin_act_volt);
```

Comment:
Future note, can we have a common place to store inversions? Imo we should assume right directions in software, have some booleans in firmware to store if values are inverted or not so its easy to tell / fix if wiring ever changes

Reply:
this is easily readable tbh. having hardware level hotfixes in the robot.cpp file is pretty minimal solution imp. the ROS topic api is assumed to be standardized and metric units.